src/pieces/commands/commands_functions.py

```diff
@@ -138,7 +136,6 @@ def list_assets(list_type_or_max='assets', **kwargs):
         else:
             print("Error: The 'Applications' object does not contain an iterable list of applications.")
 
-        print()
```

Comment:
any reason why we removed this too?

Reply:
Same thing why there is an extra line at the end of the apps and the rest of the list commands does not have that line?